pkg/azureclients/armclient/azure_armclient.go

```diff
@@ -86,10 +86,7 @@ func sender() autorest.Sender {
 				Renegotiation: tls.RenegotiateNever, // the same as default transport https://pkg.go.dev/crypto/tls#RenegotiationSupport
 			},
 		}
-		var roundTripper http.RoundTripper = transport
-		if tracing.IsEnabled() {
-			roundTripper = tracing.NewTransport(transport)
-		}
+		roundTripper := armbalancer.New(armbalancer.Options{Transport: transport})
```

Comment:
Yes, this is correct. Though we should somehow expose the `﻿Host` property, since ARM isn't always reached at the default value (management.azure.com).